Tools/px4airframes/rcout.py

```diff
@@ -50,14 +50,23 @@ def __init__(self, groups, board):
                 result +=   "# %s\n" % param.GetName()
                 result +=   "if param compare SYS_AUTOSTART %s\n" % id_val
                 result +=   "then\n"
-                result +=   "\tsh %s\n" % path
+                result +=   "\tsh /etc/init.d/airframes/%s\n" % path
+                result +=   "\tset SYS_AUTOSTART_VALUE %s\n" % id_val
                 result +=   "fi\n"
 
                 #if long_desc is not None:
                 #    result += "# %s\n" % long_desc
                 result += "\n"
 
             result += "\n"
+        result += "\n"
+        result += "if [ $SYS_AUTOSTART_VALUE = none ]\n"
+        result += "then\n"
+        result += "\techo \"ERROR  [init] No file matches SYS_AUTOSTART value found in : /etc/init.d/airframes\"\n"
+        result += "\techo \"ERROR  [init] No file matches SYS_AUTOSTART value found in : /etc/init.d/airframes\" >> $LOG_FILE\n"
+        result += "\ttone_alarm ${TUNE_ERR}\n"
+        result += "fi\n"
+        result += "unset SYS_AUTOSTART_VALUE"
```

Comment:
What happened before without this check?

Reply:
@julianoes 
I added this after we accidentally entered the wrong value and took us some time to figure it out.